src/options/options.html

```diff
@@ -148,6 +141,16 @@ <h2 data-i18n="__MSG_subtitleMisc__">Miscellaneous settings</h2>
 						</div>
 					</li>
 
+					<li>
```

Comment:
Hmm, should not you also add a `.line` css class here? (not tested, I don't know whether it works :smile:)
Or an extra div?